addons/anyuid/anyuid.addon

```diff
@@ -1,8 +1,9 @@
 # Name: anyuid
+# Var-Defaults: NAMESPACE=myproject
 # Description: Changes the default security context constraints to allow pods to run as any user.
 # Url: https://docs.okd.io/latest/creating_images/guidelines.html#openshift-origin-specific-guidelines
 
-oc adm policy add-scc-to-group anyuid system:authenticated
+oc adm policy add-scc-to-group anyuid system:authenticated -n #{NAMESPACE}
```

Comment:
@anjannath have you tried it after making this change? I mean does it now solve the issue for users?

Reply:
@praveenkumar Nope, it didn't solve the issue :(